client/gulpfile.js

```diff
@@ -197,6 +197,17 @@ gulp.task("copy:bower", function() {
     }))
 });
 
+// Copy html2canvas.
+gulp.task("copy:html2canvas", function() {
+  // There are unused Dexie files that causes trouble for uglify later
```

Comment:
Unused in what way? Could we delete those files eventually?

Reply:
This comment was copied from another location, where it's not relevant as well.
Removed both comments.